src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -1,38 +1,32 @@
+- name: Gather package facts
+  package_facts:
+
 - name: Install packages for ground base image
   block:
   - name: Install dnf plugins
-    dnf:
+    package:
       state: present
       name:
       - dnf-plugins-core
-
-  - name: Install additional repositories
-    shell: |
-      dnf config-manager --add-repo {{ item }}
-    with_items:
-    - https://cli.github.com/packages/rpm/gh-cli.repo
```

Comment:
We want this on every container, not just the development one.

Reply:
Moved to the extended package set.